src/composables/fungibleTokens.ts

```diff
@@ -66,16 +68,16 @@ export function useFungibleTokens() {
   }
 
   async function loadAvailableTokens() {
-    const response: IToken[] = camelCaseKeysDeep(await fetchAllPages(
-      () => fetchFromMiddleware('/v2/aex9?by=name&limit=100&direction=forward'),
-      fetchFromMiddleware,
-    ));
+    const tokensFetchPromises = protocolsInUse.value.map(
```

Comment:
I suggest to keep tokens grouped by protocol instead of putting them all in the same array

Reply:
OK, done. Please take a look again @martinkaintas .
Actually this change took me a lot time, but it make sense.